pkg/executor/importer/importer_testkit_test.go

```diff
@@ -311,11 +312,13 @@ func TestProcessChunkWith(t *testing.T) {
 	require.NoError(t, os.WriteFile(fileName, sourceData, 0o644))
 
 	keyspace := store.GetCodec().GetKeyspace()
+	prefixLenForOneRow := uint64(len(keyspace))
 	t.Run("file chunk", func(t *testing.T) {
 		chunkInfo := &checkpoints.ChunkCheckpoint{
 			FileMeta: mydump.SourceFileMeta{Type: mydump.SourceTypeCSV, Path: "test.csv"},
 			Chunk:    mydump.Chunk{EndOffset: int64(len(sourceData)), RowIDMax: 10000},
 		}
+		var scanedRows uint64 = 2 // EndOffset is not correct?
```

Comment:
what does `// EndOffset is not correct?` mean?

Reply:
There are 3 rows in CSV file, but only 2 rows are scanned. I doubt `Chunk.EndOffset` is not initialized correctly.

Reply:
`{Name: "skip_rows", Value: expression.NewInt64Const(1)}})`